docs/_articles/en/v3.x/scanner-commands/run.md

```diff
@@ -149,3 +149,29 @@ $ sfdx scanner:run --target "/some-project/" --severity-threshold 2
 
 ## Demo
 ![Run Example](./assets/images/run.gif)
+
+## Target Patterns
+
+In `~/.sfdx-scanner/{{ site.data.versions-v3.configfile }}`, you can specify target patterns for each engine. 
+
+If you are using ESLint and ESLint-LWC and want to avoid duplicate violations, replace `"**/*.js"` with the pattern for your LWC files to the `eslint-lwc` `targetPatterns` and add the negative pattern to the `eslint` `targetPatterns` . For example, if your LWC files are stored in a directory called `lwc`, your `{{ site.data.versions-v3.configfile }}` file might include this information:
```

Comment:
"If you are using both ESLint and ESLint-LWC simultaneously"?

Reply:
I like that, makes it more clear